omocha-api/src/main/java/org/omocha/api/interfaces/AuctionController.java

```diff
@@ -86,11 +87,15 @@ public ResponseEntity<ResultDto<Page<AuctionDto.AuctionSearchResponse>>> auction
 		@PageableDefault(page = 0, size = 10)
 		Pageable pageable
 	) {
+		Long memberId = null;
+		if (userPrincipal != null) {
+			memberId = userPrincipal.getId();
+		}
```

Comment:
```suggestion
		Long memberId = Optional.ofNullable(userPrincipal)
                        .map(UserPrincipal::getId)
                        .orElse(null);
```
기존 코드보다 Optional을 사용하는건 어떨까요

Reply:
깔끔하고 좋네요!